### ISSUE TITLE
Are Channels Implemented in the API?

### ISSUE THREAD

**Issue by verona-rupes** [irufus/gdax-java#30]
I've been reading through the gdax official documentation and they mention that we need to subscribe to channels in order to get updates. I couldn't find any channel subscriptions in the code. Is the _WebsocketFeed_ stream subscribe to a "_full channel_" ?

https://docs.gdax.com/#channels

**Comment by robevansuk** [comment]
yes its not implemented yet - I'm attempting to build a full-channel + gui on top but its relatively simple to do this sort of thing using the WebsocketFeed.

**Comment by robevansuk** [comment]
almost have a working example now. Essentially you need to process messages in sequential order. Since the websocketfeed relies on TCP based comms you should get all the messages, you just need to process them in the correct order... and timeout if the one you're waiting for next doesn't arrive in a sensible time frame.

**Comment by robevansuk** [comment]
I have a working example - it will be available soon enough. Next few days or couple of weeks max I'm hoping

**Comment by robevansuk** [comment]
I've implemented the LiveOrderbook now should be available any day now with any luck

**Comment by verona-rupes** [comment]
I kinda refactored your code for level2 and ticker channels. It was quick and dirty just to get my bot running. I realized that the connections through org.apache.tomcat.websocket.WsWebSocketContainer keeps disconnecting when using channels. Not sure why. so I switched the socket implementation to TooTallNate/Java-WebSocket. which is a standalone socket implementation and it worked fine.

**Comment by robevansuk** [comment]
ok - Full LiveOrderbook is available now over at my repo www.github.com/robevansuk/gdax-java. Fully tested and reconnects on timeouts if the message we're waiting for isn't received within 15 seconds.

**Comment by robevansuk** [comment]
I figure realtime is likely to offer much better algo trading! imaging monitoring the rate of trades coming in, seeing a spike and being able to place a trade instantly. Surely that's going to make/save more than a few %?
